components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/serialization_of_types_in_Cairo.adoc

```diff
@@ -0,0 +1,316 @@
+[id="serialization_of_types_in_Cairo"]
+= Serialization of types in Cairo
+
+When you interact with contracts, especially if you are a library or SDK developer that wants to create transactions, you need to understand how Cairo handles types that are larger than 252 bits so you can correctly formulate the calldata in a transaction.
+
+The field element (`felt252`), which contains 252 bits, is the only actual type in the Cairo VM. So all high-level Cairo types that are larger than 252 bits, such as `u256` or arrays, are ultimately represented by a list of felts.
```

Comment:
Might be a style thing for these kinds of docs, but usually only single quotes for titles. Double quotes within the text

Reply:
@Mrericsumner  These are backticks, which you use to markup code text. This is true in markdown and AsciiDoc and also on Slack (give it a try)